Sources/PostgresNIO/Connection/PostgresDatabase+PreparedQuery.swift

```diff
@@ -3,10 +3,9 @@ import Foundation
 extension PostgresDatabase {
     public func prepare(query: String) -> EventLoopFuture<PreparedQuery> {
         let name = "nio-postgres-\(UUID().uuidString)"
-        let prepare = PrepareQueryRequest(query, as: name)
-        return self.send(prepare, logger: self.logger).map { () -> (PreparedQuery) in
-            let prepared = PreparedQuery(database: self, name: name, rowDescription: prepare.rowLookupTable)
-            return prepared
+        let request = PrepareQueryRequest(query, as: name)
+        return self.send(PostgresCommands.prepareQuery(request: request), logger: self.logger).map { _ in
+            request.prepared!
```

Comment:
Are there any cases where `send()` won't set `prepared` and should we be more defensive here rather than force unwrapping?

Reply:
There are none! :) I should add this as a code comment though. This is a workaround since the send only returns a `EventLoopFuture<Void>`.